modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -158,6 +158,46 @@ Added 4 nodes, Set 8 properties, Added 4 labels
 |===
 ====
 
+[role=label--enterprise-edition label--not-on-aura label--new-5.21]
+=== Import from a Google Cloud Storage URI
+
+You can import data from a CSV file hosted in a Google Cloud Storage URI.
+
+.Import artists name and year information from a Google Cloud Storage URI
+====
+
+.gs://artists.csv
+[source, csv, filename="artists.csv"]
+----
+1,ABBA,1992
+2,Roxette,1986
+3,Europe,1979
+4,The Cardigans,1992
+----
+
+.Query
+[source, cypher, role=test-skip]
+----
+LOAD CSV FROM 'gs://artists.csv' AS row
```

Comment:
URI needs to be in the form 'gs://somebucketname/artists.csv' as the first segment of the URI path identifies the bucket to use

Noticed that the S3 docs below have a similar problem, i.e. they also need a bucket name like  's3://somebucketname/artists.csv'

Reply:
while at it, since this is also cloud storage, do you think we can conflate the two sections?

Reply:
Might be nice to - though not sure the best way to write it though 😄 

Maybe have `gs://eighties/artists.csv` with some data and `s3://nineties/artists.csv` with some different data and have the LOAD CSV parameterised with those URIs?

```
UNWIND ['gs://eighties/artists.csv', 's3://nineties/artists.csv'] AS cloudUri
LOAD CSV FROM cloudUri AS row
```

Reply:
i added a section heading instead
the UNWIND query looks cool though :)